enet server test/enet server test.cpp

```diff
@@ -2738,6 +2738,24 @@ int _tmain(int argc, _TCHAR* argv[])
 						delete p.data;
 						//enet_host_flush(server);
 					}
+                   else if (str.substr(0, 8) == "/howgay") {
```

Comment:
I haven't checked this, but I think that this is wrong.
```suggestion
                   else if (str.substr(0, 7) == "/howgay") {
```